internal/sinkcores/sentrycore/worker.go

```diff
@@ -130,6 +132,14 @@ func (w *worker) capture(errCtx *errorContext) {
 	if errCtx.Level == zapcore.WarnLevel {
 		tags["transient"] = "true"
 	}
+	// Extract the service name, if present in the fields.
+	for _, f := range errCtx.Fields {
+		if f.Key == globallogger.ResourceFieldKey {
+			if r, ok := f.Interface.(*encoders.ResourceEncoder); ok {
+				tags["service_name"] = r.Name
+			}
+		}
+	}
```

Comment:
- Would this be duplicative of `SetContext`? I'm not sure I understand the interplay here, it does seem like we can already filter by log fields.
- We should set all `Resource` fields into the tags here
- If we set `Resource` we should also set `Scope`

Reply:
oh I see we already set scope!

Reply:
> Can you? Docs says we can only search within the body and the tags.

You're right, I confused tags and the other stuff!

> That being said, it prompted me to double check and we can use * with the tags, so we can use the scope to build ownership rules (the UI really doesn't suggest so, which is what made think otherwise at first). I've set up a test so we can confirm this.

I do think it is useful to have the `Resource` fields searchable regardless!

Reply:
Can we have dots in the tag name? I'd prefer something like:

```
resource.service_name
resource.version
```

etc, just so that it aligns with the logs